lib/kafka.rb

```diff
@@ -243,8 +243,14 @@ class FailedScramAuthentication < SaslScramError
   #
   # @see Client#initialize
   # @return [Client]
-  def self.new(**options)
-    Client.new(**options)
+  def self.new(seed_brokers = nil, **options)
+    # We allow `seed_brokers` to be passed in either as a positional _or_ as a
+    # keyword argument.
+    if seed_brokers.nil?
+      Client.new(**options)
+    else
+      Client.new(seed_brokers: seed_brokers, **options)
```

Comment:
question for the future: will both of them be supported or will the non positional be deprecated?

Reply:
In the future, major release, the non-positional one will probably be deprecated and removed. I'll give it at least 6-9 months first, though.